internal/mux/muxfunc.go

```diff
@@ -8,16 +8,6 @@ import (
 // MatchFunc allows custom logic for mapping packets to an Endpoint
 type MatchFunc func([]byte) bool
 
-// MatchAll always returns true
-func MatchAll(b []byte) bool {
-	return true
-}
-
-// MatchNone always returns false
-func MatchNone(b []byte) bool {
-	return false
-}
-
```

Comment:
Why remove this and move it into test? Maybe i'm just not understanding something here...

Reply:
Just an accident on my part. Since this is all in `/internal` just wanted to clean stuff up :)